recipes/fenics-ffcx/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "fenics-ffcx-ufcx" %}
+{% set version = "0.4.2" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # PyPI sdist missing ufcx sources in 0.4.2. Should be fixed in 0.4.3
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}{{ extra }}.tar.gz
```

Comment:
You probably don't need this commented URL here.

Reply:
I kept it because that's what I plan to use starting from 0.4.3. That way I can uncomment this and remove the github URL, rather than finding the pypi url from another recipe and pasting it in again.